portfolio/src/main/webapp/index.html

```diff
@@ -53,6 +53,10 @@ <h1>Iris Liu</h1>
                             <p class="self-introduction">I'm a Chapel Hill, NC based | Student | Software Engineer | Photographer | Traveler</p>
                             <p class="self-introduction">Welcome to my personal website. I hope you find this site helpful and inspiring. And feel free to contact me anytime</p>
                         </div>
+                        <div>
+                            <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
We generally avoid setting event handlers like this directly in the HTML for several reasons, one of which is that it ties JavaScript code into the HTML, and HTML is only supposed to describe structure. (There are also more complicated security reasons.) You can set the event handler via JavaScript using addEventHandler e.g.

```
myButtonElement.addEventHandler('click', (e) => {
  getRandomQuoteUsingArrowFunctions();
});
```